packages/eslint-config-widen/src/playwright.ts

```diff
@@ -0,0 +1,16 @@
+export = {
+  overrides: [
+    {
+      extends: ['plugin:playwright/playwright-test'],
+      plugins: ['playwright'],
+      rules: {
+        // TODO: Uncomment when these rules are released
```

Comment:
Anymore info on this like a JIRA # or timeframe or just whenever we are in this file and it catches are eye?

Reply:
Not really since it's not a Widen owned project